src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -19,8 +19,8 @@
 import java.time.Instant;
 import java.util.*;
 
-@JsonIgnoreProperties(ignoreUnknown=true)
-@JsonPropertyOrder({"entry-number", "entry-timestamp", "item-hash", "key"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonPropertyOrder({"index-entry-number", "entry-number", "entry-timestamp", "item-hash", "key"})
```

Comment:
I think we should move `key` to be before `item-hash` so that it doesn't get missed when there are lots of item-hashes.

```
{"index-entry-number", "entry-number", "entry-timestamp", "key", "item-hash"}
```